docs/contributor_guide/development_process.rst

```diff
@@ -234,6 +234,35 @@ To run the same tests locally, find the corresponding pipeline definition (a
 the command to execute the tests (usually starting with
 :code:`python -m pytest`) or it refers to a template file with the command.
 
+Building the website
+^^^^^^^^^^^^^^^^^^^^
+
+The website is built using `Sphinx <https://www.sphinx-doc.org/en/master/>`_
```

Comment:
We should probably include doc build dependencies here.

Reply:
Yes, please!

I tried to use this and after installing a bunch of the sphinx extensions I still got: `TemplateNotFound('sidebar-search-bs.html')`, which was solved by installing sphinx-book-theme, but this wasn't immediately obvious from the error.

Reply:
That particular one sounds like the reason I added a pin in #740

Reply:
Added a sentence about this to point out `requirements-dev.txt`. Thanks for the suggestion!